### ISSUE TITLE
Cinnamenu@json: Menu lags when openinig and closing after resizing display scale to 125%

### ISSUE THREAD

**Issue by peachy-peaches** [linuxmint/cinnamon-spices-applets#4685]
```
 * Applet version: 5.2
 * Cinnamon version: 5.6.7
 * Distribution: Mint 21.1

 ```

**Notify author of applet**
@fredcw

**Issue**
After changing my display scale settings from 100% to 125%, Cinnamenu lags when opening and closing. It works normally otherwise, but the open/close lag is very significant. My display's resolution is 2560 x 1440 (16:9).

**Steps to reproduce**
1. Right-click on the Desktop
2. Select "Display Settings"
3. Change "Monitor Scale" from 100% to 125%
4. Click "Apply"
5. Click "Keep new configuration"
6. Click Cinnamenu icon to open menu
7. Click Cinnamenu icon to close menu

**Expected behaviour**
Cinnamenu would open and close normally without any lag

**Other information**
Display resolution is 2560 x 1440 (16:9)

**Comment by fredcw** [comment]
Hi, I can't reproduce this. How long is the delay? Is it the same each time you open the menu? What happens if you restart cinnamon (Ctrl-Alt-Esc)?

**Comment by peachy-peaches** [comment]
There's a delay of about one second from click to open/close.

https://user-images.githubusercontent.com/125409683/218986551-dc5b5a33-563f-4303-82a7-84d5302e8f55.mp4
Here's a video. The video doesn't show when I click, but it does show other types of visual delay.

**Comment by fredcw** [comment]
Yes, I see the issue now. It seems to only happen with a few themes, Orchidea being one of them. I don't think there's much I can do about it in cinnamenu as it seems to also affect other applets, though perhaps not to the same extent. So I would say that this is a performance issue with the Orchidea theme rather than cinnamenu. It may be a good idea to open an issue with the theme to see if the author can improve performance.

I notice that the theme uses more .svg files (rather than .png files) than most themes. I don't know if this is the issue but I'll look into it further to see if I can see what's causing it.

**Comment by fredcw** [comment]
I found the problem. In the file ~/.local/share/themes/Orchidea/cinnamon/cinnamon.css at line 157 which reads:

`box-shadow: 0 10px 10px rgba(0, 0, 0, 0.285), 0 6px 3px rgba(0, 0, 0, 0.345);`

This large (10px) box shadow has a big effect on performance.

I'd suggest changing this line to: 

`box-shadow: 0 6px 3px rgba(0, 0, 0, 0.2);`

Which looks pretty similar but has a big performance improvement.

Save the file and restart cinnamon.

**Comment by fredcw** [comment]
Ping @andr35

**Comment by andr35** [comment]
I've updated the theme.
Here is the pull request: https://github.com/linuxmint/cinnamon-spices-themes/pull/738

**Comment by fredcw** [comment]
@peachy-peaches If you think the problem is fixed now, could you close this issue? Thanks.
